File: DSA/Assignment 2/question1.c
#include <stdio.h>
#include <stdlib.h>
#include "sorting.h"

int main(){
	int n, arr[20];
	printf("Enter the size of array: ");
	scanf("%d",&n);

	generate(arr, n);
	printf("Before sorting array: ");
	display(arr,n);
	bubbleSort(arr,n);
	printf("After sorting array: ");
	display(arr,n);
	return 0;
}

File: Java/Assignment 6/Question1.java
import java.util.*;

class Question1{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		System.out.print("Total no of Objects to be created: ");
		int n = sc.nextInt();
		CricketPlayer cp[] = new CricketPlayer[n];

		for(int i=0; i<cp.length; i++){
			System.out.print("Enter name: ");
			String name = sc.next();
			System.out.print("Enter no of innings played: ");
			int innings = sc.nextInt();
			System.out.print("Enter no of times not out: ");
			int notout = sc.nextInt();
			System.out.print("Enter total runs: ");
			int runs = sc.nextInt();

			cp[i] = new CricketPlayer(name, innings, notout, runs);
			CricketPlayer.avg(cp[i]);
		}
		System.out.println("Before Sorting...");
		for(int i = 0; i<cp.length; i++)
			cp[i].display();

		CricketPlayer.sortPlayer(cp);
		
		System.out.println("After Sorting...");
		for(int i = 0; i<cp.length; i++)
			cp[i].display();

		sc.close();
	}

}
